tests/test_srcset.py

```diff
@@ -275,3 +275,16 @@ def test_given_aspect_ratio_and_height_srcset_signs_urls():
                                          .encode('utf-8')).hexdigest()
 
         assert(expected_signature == generated_signature)
+
+
+def test_given_fit_params_not_altered():
+    params = {'fit': 'max'}
+    _default_srcset(params)
+    assert len(params) == 1
```

Comment:
How about here instead we just test that `assert params == {'fit': 'max'}` ?
That way we can be sure that the length as well as values are the same.

Reply:
Done